scipy/_lib/tests/test__util.py

```diff
@@ -343,8 +349,9 @@ def test_contains_nan_with_strings(self):
         data4 = np.array([["1", 2], [3, np.nan]], dtype='object')
         assert _contains_nan(data4)
 
+    @pytest.mark.skip_xp_backends("jax.numpy", reason="lazy backends tested separately")
     @pytest.mark.parametrize("nan_policy", ['propagate', 'omit', 'raise'])
-    def test_array_api(self, xp, nan_policy):
+    def test_array_api_eager(self, xp, nan_policy):
```

Comment:
We should probably avoid this change, since "eager" is the default and should remain implicit - otherwise we're going to be renaming many thousands of tests, probably inconsistently.